godot-core/src/obj/traits.rs

```diff
@@ -11,6 +11,9 @@ use godot_ffi as sys;
 /// Makes `T` eligible to be managed by Godot and stored in [`Gd<T>`][crate::obj::Gd] pointers.
 ///
 /// The behavior of types implementing this trait is influenced by the associated types; check their documentation for information.
+///
+/// You wouldn't usually implement this trait yourself; see the [documentation for the
+/// `derive(GodotClass)` macro](godot_macros::GodotClass).
```

Comment:
Doc links should always point to symbols in the `godot` crate, as those are the public-facing API.

This should work because we have a reverse dev-dependency from `godot-macros` to `godot`:

https://github.com/godot-rust/gdext/blob/cccf2473000af973060d30e7d2bbdea915071f1d/godot-macros/Cargo.toml#L13-L15